ui/main.qml

```diff
@@ -301,8 +312,8 @@ StatusWindow {
     signal navigateTo(string path)
 
     function makeStatusAppActive() {
-        applicationWindow.restoreWindowState()
-        applicationWindow.visible = true
+        d.restoreWindowState()
+        applicationWindow.visibility = Window.Windowed
```

Comment:
Why this change? This will make a previously maximized window back to a normal (unmaximized) state